app/src/main/java/com/nextcloud/talk/utils/FileUtils.java

```diff
@@ -0,0 +1,46 @@
+package com.nextcloud.talk.utils;
```

Comment:
Copyright comment is missing.

Reply:
DONE :) https://github.com/nextcloud/talk-android/pull/1542/commits/4671b18b91dc4c4670438f3dd3d6689200440b32